### DISCUSSION TITLE
将刚创建的超级管理员提供给租户 页面上没找到这个地方

### DISCUSSION THREAD

**Discussion by zqb666** [zuihou/zuihou-admin-cloud#202] [02.提问] [answered]
<img width="1411" alt="image" src="https://user-images.githubusercontent.com/6092525/194735220-ee10fcbb-dd78-4c34-96dd-232e3381c309.png">

**Comment by zuihou** [comment, marked as answer]
![image](https://user-images.githubusercontent.com/8500410/194736195-bdec933b-19b4-473b-8c36-edbd9af4e944.png)


超级用户 - 切换租户 - 新增用户， 就自动完成第4步
